CHANGELOG.md

```diff
@@ -215,7 +215,7 @@
 ## 0.12.0+1
 
 * Remove the documentation link from the pubspec so this is linked to
-  pub.dartlang.org by default.
+  pub.dev by default.
```

Comment:
Why change this retroactively? At the time of the change this was true

Reply:
repo_manager does a search for outdated strings. This keeps it out of my filter.